### ISSUE TITLE
[Question] Plans for i18n

### ISSUE THREAD

**Issue by guastallaigor** [egoist/feedbackok-js#2]
Love the idea! Thinking about using it in one of mine pt-BR projects.
Do you have any plans for adding i18n or any options for changing the texts?
I'm not familiarized with preact, but I can try to help if you have any plans for this feature.
Thank you.

**Comment by egoist** [comment]
Yeah that's on the roadmap: https://github.com/egoist/feedbackok-roadmap/issues/6

It should be similar to how i18n works in React and Vue, I'll refactor the code (let me know if you can contribute a PR too) to make it easier for translation.

**Comment by guastallaigor** [comment]
Yes, I can contribute to it.
But regarding front-end frameworks, I'm more of a Vue.js developer, my React skills are not very good atm.
Anyway, let me know what I can do to help 😄
